test/integration.js

```diff
@@ -50,57 +51,59 @@ describe('integration tests', function () {
     })
   })
 
-  describe('load root package.json', function () {
-    before(function (done) {
-      this.timeout(10000)
-      // create a symlinked, and a physical copy of yargs in
-      // our fixtures directory, so that we can test that the
-      // nearest package.json is appropriately loaded.
-      cpr('./', './test/fixtures/yargs', {
-        filter: /node_modules|example|test|package\.json/
-      }, function () {
-        fs.symlinkSync(process.cwd(), './test/fixtures/yargs-symlink')
-        return done()
-      })
-    })
-
-    describe('version #', function () {
-      it('defaults to appropriate version # when yargs is installed normally', function (done) {
-        testCmd('./normal-bin.js', [ '--version' ], function (buf) {
-          buf.should.match(/9\.9\.9/)
+  if (!isWindows()) {
```

Comment:
Do we really need a new dependency for that? Why not just use `process.platform !== 'win32'`?